src/bin/pgcopydb/pgcmd.h

```diff
@@ -183,6 +183,41 @@ typedef struct ArchiveContentArray
 } ArchiveContentArray;
 
 
+/*
+ * Section options to use with pg_dump and pg_restore.
+ */
+typedef enum
+{
+	PRE_DATA = 0,
+	POST_DATA,
+} SectionOption;
+
+/*
+ * Convert SectionOption to string.
+ */
+static inline const char *
+sectionOptionToString(SectionOption section)
+{
+	switch (section)
+	{
+		case PRE_DATA:
+		{
+			return "--section=pre-data";
```

Comment:
I think I would just return "pre-data" here, and use two entries in the command line argument array:

``` c
    args[index++] = "--section";
    args[index++] = (char *) sectionOptionToString(section);
```

Reply:
makes sense 👍